test/evergreen.yml

```diff
@@ -4107,6 +4107,46 @@ buildvariants:
     - name: ".stress-test-zseries-2"
     - name: ".stress-test-zseries-3"
 
+- name: ubuntu2004-arm64
+  display_name: "~ Ubuntu 20.04 ARM64"
+  run_on:
+  - ubuntu2004-arm64
```

Comment:
It looks like `ubuntu2004-arm64` was mapped to `ubuntu2004-arm64-large` distro in Evergreen (according to the patch build). We normally run test tasks on the equivalent "small" distro, unless there's a reason to migrate to the "large" one.

Reply:
Ah right, good point. Fixed in 79632a53d010297aca12901fdf580ddc24a514ab